consistency-model.md

```diff
@@ -1,13 +1,13 @@
-Just like in SQLite, in Dqlite all SQL statements are executed within a **transaction** which is started either implicitly or explicitly. Explicit transactions are the ones started with the `BEGIN` statement. Implicit transactions are started when executing a single statement without having started an explicit transaction with `BEGIN`. Explicit transactions get committed using the `COMMIT` statement, and implicit transactions are committed automatically.
+Just like in SQLite, in Dqlite all SQL statements are executed within a *transaction* which is started either implicitly or explicitly. Explicit transactions are the ones started with the `BEGIN` statement. Implicit transactions are started when executing a single statement without having started an explicit transaction with `BEGIN`. Explicit transactions get committed using the `COMMIT` statement, and implicit transactions are committed automatically.
 
 A transaction always starts as a read transaction and gets upgraded to write transaction only once it actually starts to make changes to the database (e.g. via `INSERT` or `UPDATE` ).
 
 At any time any number of read transactions can be active, but at most one write transaction is allowed. If a read transaction tries to upgrade to write transaction while a write transaction is in progress, an error will be returned.
 
 Transactions are fully isolated and if a read transaction is started before or during a write transaction it will observe a snapshot of the database taken before the write transaction started.
 
-More precisely, Dqlite transactions are **serializable**, however their are **not linearizable**. They are serializable because Dqlite allows at most one writer at a time and readers always observe a consistent (i.e. committed) snapshot of the database, therefore transactions can't "interleave". They are not linearizable because there's no guarantee that a read transaction will always observe a snapshot of the most recently committed version of the database that existed when the read transaction started. In other words, **stale reads** are allowed.
+More precisely, Dqlite transactions can be *serialised*, however they *cannot be linearised*. They are serialisable because Dqlite allows at most one writer at a time and readers always observe a consistent (i.e. committed) snapshot of the database, therefore transactions can't "interleave". They are not linearisable because there's no guarantee that a read transaction will always observe a snapshot of the most recently committed version of the database that existed when the read transaction started. In other words, *stale reads* are allowed.
```

Comment:
serializable and linearizable (American English) are database lingo, serialised and linearised don't feel right imo.

Reply:
Seems like BE allows "serialize" as well, and "linearise/linearize" isn't really a word anyway, so I'm fine with going for the "z" variant for those two.